specs/wp-signup-spec.js

```diff
@@ -168,14 +173,48 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
 										} );
 									} );
 
-									test.describe( 'Step Eight: Can activate my account from an email', function() {
-										test.before( function() {
-											return this.emailClient = new EmailClient( signupInboxId );
+									test.describe( 'Step Eight: Can not publish until email is confirmed', function() {
+										test.it( 'Can see a disabled publish button', function() {
+											const blogPostTitle = dataHelper.randomPhrase();
+											const blogPostQuote = dataHelper.randomPhrase();
+
+											this.adminTopBar = new AdminTopBar( this.driver );
+											this.adminTopBar.createNewPost();
+
+											this.editor = new EditorPage( this.driver );
+											this.editor.enterTitle( blogPostTitle );
+											this.editor.enterContent( blogPostQuote + '\n' );
+											return this.editor.publishEnabled().then( ( enabled ) => {
+												return assert.equal( enabled, false, 'Publish button is not disabled when activation link has not been clicked' );
+											} );
 										} );
 
-										test.it( 'Can see a single activation message', function() {
-											return this.emailClient.pollEmailsByRecipient( emailAddress ).then( function( emails ) {
-												return assert.equal( emails.length, 1, 'The number of invite emails is not equal to 1' );
+										test.describe( 'Step Nine: Can activate my account from an email', function() {
+											test.before( function() {
+												return this.emailClient = new EmailClient( signupInboxId );
+											} );
+
+											test.it( 'Can see a single activation message', function() {
+												return this.emailClient.pollEmailsByRecipient( emailAddress ).then( function( emails ) {
+													inboxEmails = emails;
+													return assert.equal( inboxEmails.length, 1, 'The number of invite emails is not equal to 1' );
+												} );
+											} );
+
+											test.describe( 'Step 10: Can publish when email is confirmed', function() {
```

Comment:
This is the only step in this sequence that's labelled with the digits `10` instead of `Ten`.  Not a big deal, but worth changing for consistency.

If you're ambitious, this would be as good of a time as any to tackle https://github.com/Automattic/wp-e2e-tests/issues/356 as well